.github/MOBILE_DEPLOYMENT.md

```diff
@@ -0,0 +1,217 @@
+# Mobile Deployment Guide
```

Comment:
nit: did we want to keep this markdown readme in a hidden folder? or should we move it to the project root or create a "docs" folder